docs/_lab/03-running-commands/index.md

```diff
@@ -6,16 +6,16 @@ time: Approximately 5 minutes
 
 You can use Bolt to run arbitrary commands on a set of remote hosts. Let's see that in practice before we move on to more advanced features. Choose the exercise based on the operating system of your test nodes.
 
-- [Running shell commands on Linux nodes](#running-shell-commands-on-linux-nodes)
-- [Running PowerShell commands on Windows nodes](#running-powershell-commands-on-windows-nodes)
+- [Running shell Commands on Linux Nodes](#running-shell-commands-on-linux-nodes)
```

Comment:
I think either capitalize 'Running Shell Commands on Linux Nodes' or 'Running shell commands on Linux nodes' - shell should also be capped/uncapped with other words.